src/components/pages/Earn/index.tsx

```diff
@@ -374,8 +375,10 @@ const EarnContent: FC = () => {
             action={PageAction.Earn}
             subtitle="Ecosystem rewards with mStable"
           />
-          <MerkleClaims />
-          <PoolsOverview />
+          <MassetPage>
```

Comment:
If the goal is to show the warning here, this shouldn't be needed, since Masset forging/swaps/etc isn't involved for Earn.